sci-biology/ugene/ugene-1.15.1.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit qt4-r2
+
+DESCRIPTION="A free open-source cross-platform bioinformatics software"
+HOMEPAGE="http://ugene.unipro.ru"
+SRC_URI="http://${PN}.unipro.ru/downloads/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="cpu_flags_x86_sse2"
+
+DEPEND="
+	dev-qt/qtgui:4"
+RDEPEND="${DEPEND}"
+
+LANGS="cs en ru zh"
+
+src_configure() {
+	local CONFIG_OPTS
+	if use amd64; then
+		CONFIG_OPTS+=( CONFIG+="x64" )
```

Comment:
`CONFIG_OPTS+= CONFIG+` huh? Appending to `CONFIG`, which gets appended to `CONFIG_OPTS`, does that make sense?